packages/service/src/core/runTest.ts

```diff
@@ -46,18 +46,15 @@ export async function runTest(runTestContext: RunTestContext): Promise<InvokeRes
         }
 
         const results = await retry(
```

Comment:
Does this also need destructuring now?

```suggestion
        const { results } = await retry(
```

Reply:
It's returning the results directly. The reason I'm destructuring in the client is because I'm returning the attempt count as well -- though it's not used at the moment. I figure it could be leverage in some sort of logging.